### PR TITLE
Improve user experience of the app-sso-client fragment.

### PR DESCRIPTION
**Pull request by dmbelyaev**
- Generate additional section in the host's README describing how to use feature added by this fragment
- Add HTTPS redirectURI to the list of URIs in the generated ClientRegistration

Signed-off-by: Dmitry Belyaev <dbelyaev@vmware.com>

### CODE REVIEW

File: fragments/app-sso-client/accelerator.yaml [hunk 1]
```diff
@@ -32,3 +32,38 @@ engine:
...
+            with: |
+              '${existingContent}
+              
+              > **_NOTE:_** Below you can find additional information concerning creation of AppSSO relevant resources. 
+
+              ## Resources needed for AppSSO integration
+              
+              Please first familiarize yourself with basic concepts around using AppSSO to secure your workload on TAP following 
+              ["Securing your first Workload" tutorial](https://docs.vmware.com/en/Application-Single-Sign-On-for-VMware-Tanzu/1.0/appsso/GUID-app-operators-tutorials-index.html)
+              .
```
Comment (candicenonsense) [inline fragments/app-sso-client/accelerator.yaml:49]: we have a new docs location inside the tap hierarchy for tap 1.3 - https://docs-staging.vmware.com/en/draft/VMware-Tanzu-Application-Platform/1.3/tap/GUID-app-sso-app-operators-tutorials-index.html

File: fragments/app-sso-client/config/client-registration.yaml [hunk 2]
```diff
@@ -13,5 +13,6 @@ spec:
   requireUserConsent: false
   redirectURIs:
     - "http://<my-workload-hostname>/login/oauth2/code/<my-workload>"
+    - "https://<my-workload-hostname>/login/oauth2/code/<my-workload>"
   scopes:
```
Comment (candicenonsense) [inline fragments/app-sso-client/config/client-registration.yaml:17]: Could this be a setting?  I can definitely see a need for one or the other, not necessarily both.

File: fragments/app-sso-client/accelerator.yaml (hunk 1 above)
Comment (trisberg) [reply fragments/app-sso-client/accelerator.yaml:49]: would that URL then become -  https://docs.vmware.com/en/VMware-Tanzu-Application-Platform/1.3/tap/GUID-app-sso-app-operators-tutorials-index.html once docs are published?

Comment (candicenonsense) [reply fragments/app-sso-client/accelerator.yaml:49]: yes, I think so.

Comment (dmbelyaev) [reply fragments/app-sso-client/accelerator.yaml:51]: I've changed the link, although the new one is not currently available.

File: fragments/app-sso-client/config/client-registration.yaml (hunk 2 above)
Comment (dmbelyaev) [reply fragments/app-sso-client/config/client-registration.yaml:16]: Done

Comment (candicenonsense) [reply fragments/app-sso-client/config/client-registration.yaml:16]: This is all looking great

**Review by candicenonsense — APPROVED** [review]

**Pull request merged**